xbmc/cores/AudioEngine/AESinkFactory.cpp

```diff
@@ -85,12 +85,15 @@ IAESink *CAESinkFactory::Create(std::string &device, AEAudioFormat &desiredForma
   return nullptr;
 }
 
-void CAESinkFactory::EnumerateEx(std::vector<AESinkInfo> &list, bool force)
+void CAESinkFactory::EnumerateEx(std::vector<AESinkInfo> &list, bool force, std::string driver)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types